git-branchless/src/commands/reword.rs

```diff
@@ -3,11 +3,15 @@
 use lib::core::check_out::CheckOutCommitOptions;
 use lib::core::repo_ext::RepoExt;
 use rayon::ThreadPoolBuilder;
-use std::collections::HashMap;
+#[allow(unused_imports)]
```

Comment:
Can we remove the `#[allow(unused_imports)]` here?

Reply:
Yes, done. Sorry; I misunderstood the unused import warning after adding `BTreeMap` to the tests. I've moved it into the `mod tests { ... }` and now everyone's happy.

Reply:
... moved the import of `BTreeMap`, that is